### ISSUE TITLE
if a clone task need more than 24 hours to finish. it will never be done?

### ISSUE THREAD

**Issue by suhiymof** [MsGsuite/CloneBot_Heroku#38]
If a clone task need more than 24 hours to finish. 
it would never be done? 
Because of the bot will restart every 24h. And the sa file and favourite drive list would lost?

**Comment by TheCaduceus** [comment]
> If a clone task need more than 24 hours to finish. it would never be done? Because of the bot will restart every 24h. And the sa file and favourite drive list would lost?

Yes it will never if you deployed it on Heroku, because Heroku force restart all deployed apps after 24 Hours! You can use other Services like Okteto still.

**Comment by missemily2022** [comment]
@suhiymof You can use Okteto as a Alternative or Railway.app with some tweaks

**Comment by suhiymof** [comment]
> tweaks

ok， thx， I'll try it.

**Comment by suhiymof** [comment]
> > If a clone task need more than 24 hours to finish. it would never be done? Because of the bot will restart every 24h. And the sa file and favourite drive list would lost?
> 
> Yes it will never if you deployed it on Heroku, because Heroku force restart all deployed apps after 24 Hours! You can use other Services like Okteto still.
ok， thx， I'll try it.

**Comment by suhiymof** [comment]
> > If a clone task need more than 24 hours to finish. it would never be done? Because of the bot will restart every 24h. And the sa file and favourite drive list would lost?
> 
> Yes it will never if you deployed it on Heroku, because Heroku force restart all deployed apps after 24 Hours! You can use other Services like Okteto still.

ok， thx， I'll try it.

**Comment by suhiymof** [comment]
> @suhiymof You can use Okteto as a Alternative or Railway.app with some tweaks

ok， thx， I'll try it.
